src/main/java/flab/delideli/encrypt/Encryption.java

```diff
@@ -1,17 +1,24 @@
 package flab.delideli.encrypt;
 
+import lombok.SneakyThrows;
 import org.springframework.stereotype.Component;
 
 import java.security.MessageDigest;
+import flab.delideli.controller.ExceptionAdvice;
 import java.security.NoSuchAlgorithmException;
 
 @Component
-public class EncryptPassword {
+public class Encryption {
 
-    public String encrypt(String password) throws NoSuchAlgorithmException {
-        MessageDigest md = MessageDigest.getInstance("SHA-256");
+    public String encrypt(String password) {
+        MessageDigest md = null;
+        try {
+            md = MessageDigest.getInstance("SHA-2561111");
+        }
+        catch (NoSuchAlgorithmException e) {
+            throw new RuntimeException("잘못된 암호화 알고리즘이 들어갔습니다.",e);
```

Comment:
이 메소드를 사용하는 사용자가 납득할 수 있는 말을 쓰는게 좋습니다. `잘못된 암호화 알고리즘이 들어갔습니다.` 라는 말은 사용자가 "어? 내가 값을 잘못넣었나?" 하는 착오를 일으킬 수 있으니까요?

```suggestion
            throw new RuntimeException("잘못된 암호화 알고리즘이 들어갔습니다.", e);
```

Reply:
네, 알맞는 말을 찾을 수 있도록 생각해보고 수정하겠습니다.